src/content/en/tools/chrome-devtools/network-performance/reference.md

```diff
@@ -5,11 +5,9 @@ description: A comprehensive reference of Chrome DevTools Network panel features
 {# wf_updated_on: 2016-12-22 #}
 {# wf_published_on: 2015-04-13 #}
 
+{% include "web/tools/chrome-devtools/_shared/styles.html %}
```

Comment:
I merged this without noticing the single quote :( I'll fix in master and see about adding a test.